charts/rancher-monitoring/101.1.0+up40.1.2/templates/alertmanager/servicemonitor.yaml

```diff
@@ -37,6 +37,18 @@ spec:
 {{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
     metricRelabelings:
 {{ tpl (toYaml .Values.alertmanager.serviceMonitor.metricRelabelings | indent 6) . }}
+    {{ if .Values.global.cattle.clusterId }}
```

Comment:
same comment as above regarding moving `metrics-relabeling` outside the `if` statement.